.travis.yml

```diff
@@ -69,7 +69,7 @@ matrix:
 #           env: MAIN_CMD='pep8' SETUP_CMD='--count py/desiutil'
 
 before_install:
-    - curl ipinfo.io
+    - curl ipinfo.io 
```

Comment:
It now looks like you're only changing .travis.yml by adding a whitespace to the end of a line.  Please revert .travis.yml if this is really the only change.